turret_control:
  ros__parameters:
    lower_angle_deg: -10.0
    upper_angle_deg:  10.0
    pub_period_s: 0.02
    ctrl_period_s: 0.01
    pins:
      - 17
      - 18
      - 27
      - 22
    step_count_per_rot: 4096
